ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/templates/Template1.kt

```diff
@@ -117,6 +117,7 @@ private fun ColumnScope.Template1MainContent(state: PaywallViewState.Loaded) {
         IntroEligibilityStateView(
             textWithNoIntroOffer = localizedConfig.offerDetails,
             textWithIntroOffer = localizedConfig.offerDetailsWithIntroOffer,
+            textWithMultipleIntroOffers = localizedConfig.offerDetailsWithMultipleIntroOffers,
```

Comment:
I wonder if we should change to just pass the `ProcessedLocalizedConfig` instead to avoid these 3 parameters...

Reply:
The thing is this view can be used for `offerDetails*` or for `callToAction*`. See `PurchaseButton`.

Reply:
Ahhh true I forgot, nvm